Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'charlock_holmes', '~> 0.6.9.4'
 gem 'dynamic_form', '~> 1.1.4'
 gem 'exception_notification', '~> 3.0.1'
 gem 'fancybox-rails', '~> 0.2.1'
-gem 'foundation-rails', '~> 5.2.1.0'
+gem 'foundation-rails', '~> 5.5.3.2'
```

Comment:
For much the same reason as not moving to Bootstrap 3, Foundation 6 is a whole new framework (and as I had to look that up, I should add a note)